apps/bfd-data-npi/src/test/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookupTest.java

```diff
@@ -1,61 +1,57 @@
 package gov.cms.bfd.data.npi.lookup;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.ByteArrayInputStream;
 import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
 import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 /** NPI org lookup test. */
 public class NPIOrgLookupTest {
 
+  /** Global variable for NPILookup. */
+  NPIOrgLookup npiOrgDataLookup;
+
+  /** Global variable for npiOrgDisplay. */
+  Optional<String> npiOrgDisplay;
+
+  /** Setup Before Each test method. */
+  @BeforeEach
+  void setup() throws IOException {
+    npiOrgDataLookup = NPIOrgLookup.createNpiOrgLookupForTesting();
+    npiOrgDisplay = Optional.empty();
+  }
+
   /** Return Fake NPI Org Data when the parameter bfdServer.include.fake.drug.code is true. */
   @Test
   public void shouldReturnFakeOrgDataWhenConstructorSetToTrue() throws IOException {
-    NPIOrgLookup npiOrgDataLookup = NPIOrgLookup.createNpiOrgLookupForTesting();
-    Optional<String> npiOrgDisplay =
+    npiOrgDisplay =
         npiOrgDataLookup.retrieveNPIOrgDisplay(Optional.of(NPIOrgLookup.FAKE_NPI_NUMBER));
     assertNotEquals(null, npiOrgDisplay.get());
   }
```

Comment:
Should fix this text name. (and others in this class that are no longer accurate)